modules/util/pwd_complexity.go

```diff
@@ -0,0 +1,59 @@
+package util
+
+import (
+	"regexp"
+)
+
+//Complexity struct for Complexity checks
+type Complexity struct {
+	Length         int
+	ContainsUpper  bool
+	ContainsLower  bool
+	ContainsDigist bool
+	ContainsSpec   bool
+	UpperCount     int
+	LowerCount     int
+	SpecCount      int
+	DigitCount     int
+	Score          int
+}
+
+//CheckComplexity function check pwd
+func CheckComplexity(pwd *string) Complexity {
+	var KindSymbols = make(map[string]string)
+	KindSymbols["lower"] = "[a-z]"
+	KindSymbols["upper"] = "[A-Z]"
+	KindSymbols["digits"] = "[0-9]"
+	KindSymbols["spec"] = `[\!\@\#\$\%\^\&\*\(\\\)\-_\=\+\,\.\?\/\:\;\{\}\[\]~]`
+	var c Complexity
+	c.Length = len(*pwd)
+
+	MatchLower := regexp.MustCompile(KindSymbols["lower"])
```

Comment:
Can you move this to a package level variable rather than it being compiled every single time ?